starter-code/app.js

```diff
@@ -107,11 +104,16 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function multiplyArray(multArr) { //eslint-disable-line
-
+  multArr = testArray;
```

Comment:
Make sure you only use multArr (or whatever is being provided as an argument) so the function can use any array and not just the test array.